Add extra prerequisite in order to develop under windows (#1060)
tauri-apps/tauri-docs: docs/guides/getting-started/prerequisites.md

winget install --id Rustlang.Rustup
```

:::caution MSVC toolchain as default

> For full support for Tauri and tools like [`trunk`] make sure the MSVC Rust toolchain is the selected `default host triple` in the installer dialog.<br/>Depending on your system it should be either `x86_64-pc-windows-msvc`, `i686-pc-windows-msvc`, or `aarch64-pc-windows-msvc`.

Reviewer: Is trunk supposed to be a link here? If not then remove the brackets around it. I'd also remove the `<br/>`, I think it's fine to be one paragraph.

Reviewer 2: Yeah it was supposed to be a link but i forgot to add that to my review comment...

> For full support for Tauri and tools like [`trunk`] make sure the MSVC Rust toolchain is the selected `default host triple` in the installer dialog.<br/>Depending on your system it should be either `x86_64-pc-windows-msvc`, `i686-pc-windows-msvc`, or `aarch64-pc-windows-msvc`.

Reviewer: ```suggestion
For full support for Tauri and tools like [`trunk`] make sure the MSVC Rust toolchain is the selected `default host triple` in the installer dialog. Depending on your system it should be either `x86_64-pc-windows-msvc`, `i686-pc-windows-msvc`, or `aarch64-pc-windows-msvc`.
```

> If you already had Rust installed, you can make sure the correct toolchain is installed by running this command:

Reviewer: ```suggestion
If you already have Rust installed, you can make sure the correct toolchain is installed by running this command:
```

Reviewer 2: i wanted to make clear that this means, "if you had it installed prior to reading this guide". Does this still come across using `have`? (i should've worked harder in school 😞)

Reviewer: They convey the same thing, just different tenses. "have" is present tense (would which we prefer overall for writing style). Since you have "already" in front of it then that conveys it was installed prior to reading.

Reviewer 2: Okay, thank you for the explanation ❤️

winget install --id Rustlang.Rustup
```

> #### 3. MSVC toolchain as default
> 
> In order to be able to install tools like `tauri-cli` and `trunk` you need to make sure that the proper toolchain is set as default.
> 
> To display the current configured default toolchain please run the following command:
> 
> ```powershell
> rustup default
> ```
> 
> The command should return either `i686-pc-windows-msvc`, `x86_64-pc-windows-msvc`, or `aarch64-pc-windows-msvc`.
> 
> If the command returns a different toolchain, please configure the default toolchain in the following way: 
> 
> ```powershell
> rustup default stable-msvc
> ```
> 

Reviewer 2: To make it a bit more clear what direction i thought we should go here:
`````suggestion
:::caution MSVC toolchain as default

For full support for Tauri and tools like [`trunk`] make sure the MSVC Rust toolchain is the selected `default host triple` in the installer dialog.<br/>Depending on your system it should be either `x86_64-pc-windows-msvc`, `i686-pc-windows-msvc`, or `aarch64-pc-windows-msvc`.

If you already had Rust installed, you can make sure the correct toolchain is installed by running this command:

```powershell
rustup default stable-msvc
```

:::
`````

This way we could catch both cases:
- the installer selected gnu for whatever reason (user had mingw installed but not msvc or something)
- the user already had rust installed, but with the wrong toolchain

P.S. no need to accept my suggestion as-is, just wanted to visualize it.
